core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/setoperators/UnionOperator.kt

```diff
@@ -0,0 +1,13 @@
+package ch.ergon.dope.resolvable.clause.model.setoperators
+
+import ch.ergon.dope.resolvable.clause.ISelectOffsetClause
+
+class UnionOperator(
+    leftSelect: ISelectOffsetClause,
+    rightSelect: ISelectOffsetClause,
+    duplicatesAllowed: Boolean = false,
```

Comment:
would remove this default value (only used once)

Reply:
done